source/1_engine/rex_engine/src/filesystem/vfs.cpp

```diff
@@ -288,11 +298,18 @@ namespace rex
 
     void init(rsl::string_view root)
     {
-      if(rsl::optional<rsl::string_view> arg = cmdline::get_argument("Root"))
-      {
-        root = arg.value();
-      }
+      g_is_initialized  = true;
+      g_keep_processing = true;
```

Comment:
INFO: Would there be a better solution than keeping track of booleans?

Reply:
Yes, probably, but not part of this review

Reply:
Can you make an issue for it please?